plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -209,20 +242,42 @@ def eval_deisolate(self, port_name):
         self.ports_states.pop(port_name)
         self.logger.warning(f"Deisolated port: {port_name}. dry_run: {self.dry_run}")
                 
-    def get_rate_and_update(self, port_name, counter_name, new_val):
-        port_data = self.ports_data.get(port_name)
-        if port_data:
-            old_val = port_data.get(counter_name)
+    def get_rate_and_update(self, port_name, counter_name, new_val, timestamp):
+        port_obj = self.ports_data.get(port_name)
+        if port_obj:
+            if timestamp - port_obj.last_timestamp < 1:
```

Comment:
timestamp is on what size? seconds? milliseconds? why are you return the counter name and do not change the value if the timestamp is too low? how is it too low?

Reply:
it can get updated already (since it's rate) in a previous calculation so we're avoiding double calculations